tests/integration_tests/security/test_nv.py

```diff
@@ -16,31 +16,8 @@
 start providing the feature by mistake.
 """
 
-import pytest
 
-
-@pytest.fixture
-def uvm_with_cpu_template(microvm_factory, guest_kernel, rootfs, cpu_template_any):
-    """A microvm fixture parametrized with all possible templates"""
-    vm = microvm_factory.build(guest_kernel, rootfs)
-    vm.spawn()
-    cpu_template = None
-    if isinstance(cpu_template_any, str):
-        cpu_template = cpu_template_any
-    vm.basic_config(cpu_template=cpu_template)
-    if isinstance(cpu_template_any, dict):
-        vm.api.cpu_config.put(**cpu_template_any["template"])
-    vm.add_net_iface()
-    vm.start()
-    yield vm
-
-
-def test_no_nv_when_using_cpu_templates(uvm_with_cpu_template):
-    """
-    Double-check that guests using CPU templates don't have Nested Virtualization
-    enabled.
-    """
-
-    vm = uvm_with_cpu_template
-    rc, _, _ = vm.ssh.run("[ ! -e /dev/kvm ]")
+def test_no_nv(uvm_any_booted):
```

Comment:
out of scope: the name of this test is so cryptic, I had to read the code to understand what was happening lol

Reply:
Sorry that was my fault. I wouldn't be against giving it a more proper name.

Reply:
Done